cognite/neat/rules/models/asset/_rules.py

```diff
@@ -0,0 +1,155 @@
+import sys
+from typing import TYPE_CHECKING, Any, Literal, cast
+
+from pydantic import Field, field_validator, model_validator
+from pydantic.main import IncEx
+from rdflib import Namespace
+
+from cognite.neat.constants import PREFIXES
+from cognite.neat.issues import MultiValueError
+from cognite.neat.rules import issues
+from cognite.neat.rules.models._base import BaseRules, SheetList
+from cognite.neat.rules.models.domain import DomainRules
+from cognite.neat.rules.models.entities import (
+    CdfResourceEntityList,
+    ClassEntity,
+    MultiValueTypeInfo,
+    ParentClassEntity,
+    Undefined,
+)
+from cognite.neat.rules.models.information import (
+    InformationClass,
+    InformationMetadata,
+    InformationProperty,
+    InformationRules,
+)
+
+if TYPE_CHECKING:
+    from cognite.neat.rules.models.dms._rules import DMSRules
+
+
+if sys.version_info >= (3, 11):
+    from typing import Self
+else:
+    from typing_extensions import Self
+
+
+class AssetMetadata(InformationMetadata): ...
```

Comment:
Do we need a new role type?
```suggestion
class AssetMetadata(InformationMetadata):
    role: ClassVar[RoleTypes] = RoleTypes.asset
```

Reply:
Very good point! Missed it yesterday !